### DISCUSSION TITLE
Bond filter don't work

### DISCUSSION THREAD

**Discussion by caijue0** [peridigm/peridigm#308] [Q&A]
Hello,
When I added the bond filter to the input file, no pre-cracks were generated properly. This is my yaml file, and the model was generated by cubit. please tell me what I am doing wrong.

[tensile.txt](https://github.com/peridigm/peridigm/files/11028579/tensile.txt)

![image](https://user-images.githubusercontent.com/108057964/226614569-b6324b91-33e3-42a2-a87b-5a6a020c2482.png)

**Comment by IntePNum** [comment]
I don't know!!!why
